plugins/extractors/bigquery/testdata/data.yaml

```diff
@@ -66,6 +66,22 @@ projects:
               birthday: "1992-05-01"
               skillNum: 11
               created_at: '2022-01-05T12:00:00'
+        - id: table_b
+          columns:
+            - name: id
+              type: INTEGER
+              mode: REQUIRED
+            - name: name
+              type: STRING
+              mode: REQUIRED
+          data:
+            - id: 1
+              name: alice
+            - id: 2
+              name: bob
+          metadata:
+            - type: VIEW
+              view_query: select id,name from test-project-id.dataset1.table_a
```

Comment:
i don't think it is supported by the emulator, 
currently it is hardcoded to `TABLE` [here](https://github.com/goccy/bigquery-emulator/blob/8ccde288d63846122e085433e0a03edfadba361c/types/types.go#L35-L57), we can skip this test for now if it's not feasible

Reply:
@haveiss Yes, as seen VIEW is not supported in table metadata in BQ emulator, hence not able to test this through current flow.
I'll update the code